packages/keyda-demo/src/lib/Form/index.js

```diff
@@ -1,21 +1,16 @@
-import React, { useCallback } from 'react';
+import React from 'react';
 
-const Form = (props) => {
-  const { onSubmit, children, ...rest } = props;
-
-  const handleSubmit = useCallback((e) => {
-    e.preventDefault();
-    onSubmit();
-    console.log('inside of the library!');
-  }, []);
+import FormToSubmit from '../FormToSubmit';
+import { KeyStateProvider } from '../Context';
 
+const Form = (props) => {
   return (
     <React.Fragment>
-      <form onSubmit={handleSubmit} {...rest}>
-        {children}
-      </form>
+      <KeyStateProvider>
+        <FormToSubmit {...props} />
+      </KeyStateProvider>
     </React.Fragment>
   );
 };
```

Comment:
To use the context provider, I made another child component located below this `Form` component.